### ISSUE TITLE
Add groupId filtering for demo_key reviews

### ISSUE THREAD

**Issue by ck1125** [reevoo/reevoo.github.io#49]
Currently working on integrating with the API and unsure about how to make calls that retrieve all reviews for all products in a given groupId for an organization. Please modify the documents to show how this works and what the response data looks like.

Thanks.

**Comment by lpil** [comment]
Hi Eli!

I'm afraid our API does not currently provide this feature. Is there anything else we can help you with?

Thanks,
Louis

**Comment by Antti** [comment]
@ck1125 Can you elaborate, what is groupId? Is it our product_group_id?

**Comment by ck1125** [comment]
Yes  the product_group_id. Apologies about the wrong terminology and late response.
